assets/javascripts/kitten/components/cards/reward-card/stories.js

```diff
@@ -356,7 +379,7 @@ export const RewardCard = () => {
                     </ExpandBoard>
                   )}
                 </RewardCardComponent.RowContent>
-                {withImage && (
+                {visual !== null && (
```

Comment:
Normalement 
```suggestion
                {visual && (
```
devrait suffire. `null`, `false` et `undefined` sont falsy :).